visualization/test/test-page/App.jsx

```diff
@@ -107,6 +107,7 @@ import {largeFeatureCount} from '../__mock_data/largeFeatureCount';
                         }}
                         requestPredictions={this.generateRandomProbs.bind(this, classDimension)}
                         stringParams={{contextualHelp: this.messages}}
+                        // theme={"dark"}
```

Comment:
What's the plan here?

Reply:
its just a test file... Ideally we'd have all the theming and settings togglable in the  wrapper to allow easily changing these configs to see the result. That is what I should do.